src/comms/config_layer.hpp

```diff
@@ -26,8 +26,8 @@
 // lastly, stores bytes of subsec_sizes array
 #define CONFIG_PATH "/config.pack"
 
-// define CONFIG_OFF_ROBOT macro when running off of real robot (testing firmware away from actual robot)
-// #define CONFIG_OFF_ROBOT 
+// define DISREGARD_REF_CONFIG_CHECK macro when running off of real robot (testing firmware away from actual robot)
+#define DISREGARD_REF_CONFIG_CHECK 
```

Comment:
`IGNORE_REF_CONFIG_SAFETY` or something similar with the word `SAFETY` in it sounds scarier, making it clear that disabling it has safety implications. Or `DISABLE` as well to be clearer that this is also disabling ref elsewhere (if I understand correctly).